views/app.html

```diff
@@ -20,7 +20,7 @@
   <link rel="canonical" href="https://webmaker.org/{{ localeInfo.lang }}{{ currentPath }}" />
   <meta property="description" content="{{ gettext("Strips away complexity") }}" />
   <!-- /sharing -->
-
+  <link href='http://fonts.googleapis.com/css?family=Fira+Sans:400,700' rel='stylesheet' type='text/css'>
```

Comment:
Fira fonts are already in https://github.com/mozilla/webmaker.org/tree/master/public/resources/fonts and imported in less